inference-engine/src/gna_plugin/backend/am_intel_dnn.cpp

```diff
@@ -248,6 +249,16 @@ void GNAPluginNS::backend::AMIntelDNN::InitConvolutional2DComponentPrivate(intel
     ptr_inputs = &comp.ptr_inputs;
     ptr_outputs = &comp.ptr_outputs;
 }
+
+bool GNAPluginNS::backend::AMIntelDNN::isOperationCnnLegacySpecific(const Gna2Operation& op) {
+    // GNA compile target GNA_TARGET_3_0 does not support pooling window < pooling stride
+    return op.Type == Gna2OperationTypeConvolution &&
+        op.NumberOfParameters > (std::max)(PoolStrideParamIdx, PoolWinParamIdx) &&
```

Comment:
why do you use parentheses around std::max here?

Reply:
I did out of (bad) habit (see https://stackoverflow.com/questions/5004858/why-is-stdmin-failing-when-windows-h-is-included).
I will remove.